README.md

```diff
@@ -4,44 +4,41 @@
-    configurations.
-  * Version, manage, and deploy with your favorite tools and workflows that
-    you’re using with your clusters today.
-
-* **Publish custom infrastructure resources for your applications to use**
-  * Define, compose, and publish your own [infrastructure resources] with
-    declarative YAML, resulting in your own infrastructure CRDs being added to
-    the Kubernetes API for applications to use.
-  * Hide infrastructure complexity and include policy guardrails, so
-    applications can easily and safely consume the infrastructure they need,
-    using any tool that works with the Kubernetes API.
-  * Consume infrastructure resources alongside any Kubernetes application to
-    provision and manage the cloud services they need with Crossplane as an
-    add-on to any Kubernetes cluster.
-
-* **Deploy applications using a team-centric approach with OAM**
-  * Define cloud native applications and the infrastructure they require with
-    the Open Application Model ([OAM]).
-  * Collaborate with a team-centric approach with a strong separation of
-    concerns:
-    * Infrastructure operators - provide infrastructure and services for
-      applications to consume
-    * Application developers - build application components independent of
-      infrastructure
-    * Application operators - compose, deploy, and run application
-      configurations
-  * Deploy application configurations from app delivery pipelines or GitOps
-    workflows, using the proven Kubernetes declarative model.
+* **Compose Custom APIs**
+  * Crossplane allows users to compose the granular resources offered by
+    providers into higher-level abstractions using only the Kubernetes API.
+    Multiple compositions of granular resources may be mapped to a single
+    abstract type. These mappings allow infrastructure operators to enforce best
+    practices, provide a consistent API regardless of underlying infrastructure,
+    and only expose fields to application developers that they should be able to
+    tune for their use-case.
+
+* **Package and Distribute Custom APIs**
+  * Crossplane allows users to bundle their custom APIs into `Configuration`
+    packages, which specify dependencies on providers and how their granular
+    types are mapped to the higher-level composed types. These packages can be
+    pushed to any OCI registry and installed into any Kubernetes cluster where
+    Crossplane is running.
+
+* **Expose Custom APIs**
+  * Crossplane standardizes on the Kubernetes RBAC permission model, meaning
+    that custom APIs can be exposed to developers on a per-object basis using
+    familiar Kubernetes APIs, such as `Role` and `ClusterRole`. Combining this
+    permission model with Crossplane's composition functionality provides the
+    ability to define access for developers at the same level of abstraction
+    they are exposed to.
+
+* **Extend the Kubernetes API with New Providers**
```

Comment:
another good addition to this list is the XRM in general and its role in providing a universal/consistent API across multiple vendors and providers.  here's some prose from the draft CNCF proposal on that topic that we should consider including here:

> While bringing this diverse set of resources into the Kubernetes based control plane, Crossplane recognized the need for a single consistent API across all of their vendors and providers. To this end, we have created the Crossplane Resource Model (XRM).  XRM extends the Kubernetes Resource Model (KRM) in an opinionated way, resulting in a universal experience for managing resources external to the control plane itself. While interacting with the XRM, things like credentials, workload identity, connection secrets, status conditions, deletion policies, and references to other resources work the same no matter which provider they are backed by.

Reply:
This is great! It would be awesome if we could link to a longer document on XRM from here also 👍